cli/lib/cypress.js

```diff
@@ -50,6 +51,22 @@ const cypressModuleApi = {
       })
     })
   },
+
+  cli: {
+    /**
+     * Parses CLI arguments into an object that you can pass to "cypress.run"
```

Comment:
this is duplicated sorta with the comment in  `cli/lib/cli.js`. maybe the internal comment can be shorter than this

Reply:
I am now doing over-communication for the TS safe. It is nice to see intellisense for an object, and then when you try to use a method inside, see more examples.